var/spack/repos/builtin/packages/py-cartopy/package.py

```diff
@@ -0,0 +1,63 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+class PyCartopy(PythonPackage):
+    """Cartopy - a cartographic python library with matplotlib support."""
+
+    homepage = "http://scitools.org.uk/cartopy/"
+    url      = "https://github.com/SciTools/cartopy/archive/v0.16.0.tar.gz"
+
+    version('0.16.0', 'f9e2e528d7758da7c64f824548a53f32')
+
+    depends_on('py-setuptools@0.7.2:', type='build')
+    depends_on('py-cython@0.15.1:', type=('build', 'run'))
+    depends_on('py-numpy@1.10.0:',  type=('build', 'run'))
```

Comment:
Shouldn't this be 1.6+?

Reply:
at least not according to [this](https://github.com/SciTools/cartopy/blob/d07edb5e624ff3170bc139fe3b937ecffdbe4fa4/setup.py#L43), this was some time ago, but iirc I ran into this (though I can't think of why I would want to build it with an old numpy... so maybe I'm fantasising)

Reply:
That's the develop version, not the version in your package.